backend/python/app/services/implementations/story_service.py

```diff
@@ -97,18 +109,24 @@ def create_translation(self, translation):
             raise error
         return new_story_translation
 
-    def get_story_translations(self, user_id, translator, language, level):
+    def get_story_translations(self, user_id, is_translator, language, level, stage):
```

Comment:
when in the frontend is `stage` getting passed into this service?

Reply:
oops it is not being used i'll remove it--i was originally planning to use it for the backend checks but then decided it makes more calls to the db than necessary so made separate helper functions for it